src/libravi/RaviSession.ts

```diff
@@ -22,6 +22,25 @@ export enum RaviSessionStates {
   CLOSED = "closed"
 };
 
+export interface RaviSessionParams {
+  /**
+   * The minimum jitter buffer duration.
+   * 
+   * The lower the jitter buffer duration, the lower the round-trip audio latency.
```

Comment:
For the minimum jitter buffer duration, I'm concerned that the comments about "setting it too low" will confuse people. Let's change this comment to something like: 

"The minimum jitter buffer duration (in seconds). In practice, this should always be set to 0 (the default). Increasing the minimum jitter buffer means that all audio will always be buffered at least by this amount of time, which is rarely desirable. You may, however, want to set the maximum jitter buffer duration if you're increasing frequent audio drop-outs; refer to audioMaxJitterBufferDuration for more details."